src/memspaces/memspace_numa.h

```diff
@@ -27,6 +27,8 @@ extern "C" {
 umf_result_t umfMemspaceCreateFromNumaArray(size_t *nodeIds, size_t numIds,
                                             umf_memspace_handle_t *hMemspace);
 
+extern umf_memspace_handle_t UMF_MEMSPACE_HOST;
```

Comment:
Hm, I think for memspaces we'll need to follow the same approach as with ops - i.e. create a function that returns memspace handle instead of defining a global variable. See: https://github.com/oneapi-src/unified-memory-framework/pull/201

@vinser52 @bratpiorka @pbalcer are we OK with having predefined memspaces as functions? I'm just wondering if we should have a function per memspace or just a single function that accepts some enum (.e.g. `umfMemspaceGet(UMF_MEMSPACE_HOST)`)?